Makefile.am

```diff
@@ -112,7 +112,8 @@ run-root-ipython: all
 	sudo env GI_TYPELIB_PATH=${GIDIR} LD_LIBRARY_PATH=${LIBDIRS} PYTHONPATH=src/python G_MESSAGES_DEBUG=all LIBBLOCKDEV_CONFIG_DIR=data/conf.d/ i${PYTHON}
 
 pylint:
-	which $(PYLINT) && $(PYLINT) -E src/python/gi/overrides/BlockDev.py || echo "pylint not available, skipping"
+	which $(PYLINT) || ( echo "pylint not available, skipping"; exit 0 )
```

Comment:
Doesn't the `exit` only exit the subshell?

Reply:
Fixed.